config/owasp/suppressions.xml

```diff
@@ -19,41 +19,9 @@
   <!--Please add all the temporary suppression under the below section-->
```

Comment:
@kiran-yenigala-hmcts we can have another section for false positives like the temporary section. Please refer data-store suppression file.

Reply:
done